rpcd/patches/openstack-hosts-u-suk-dev-issues-1629.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/playbooks/roles/openstack_hosts/defaults/main.yml b/playbooks/roles/openstack_hosts/defaults/main.yml
```

Comment:
I hate it too, but kilo. Forking would encourage ppl to do the wrong thing and patch things there. 

Having an ugly solution is maybe a good solution in some cases.